src/node_jsvmapi_types.h

```diff
@@ -19,7 +21,10 @@ enum napi_property_attributes {
   napi_default = 0,
   napi_read_only = 1 << 0,
   napi_dont_enum = 1 << 1,
-  napi_dont_delete = 1 << 2
+  napi_dont_delete = 1 << 2,
+
+  // For internal use only by the NAPI C++ wrapper.
+  napi_instance_property = 1 << 10,
```

Comment:
Agreed. I didn't like doing this at the time.

Now, I'm actually thinking of keeping this as part of the public C API (actually switching it to `napi_static_property`) and adding support for it in `napi_define_class()` (formerly `napi_create_constructor()`). The purpose is to allow having a single array of `napi_property_descriptor`s that are passed to a single call to `napi_define_class()`, for defining both instance and static properties & methods.